tests/switchmap_/core/test_rest.py

```
@@ -0,0 +1,504 @@
+#!/usr/bin/env python3
+"""Comprehensive tests for rest.py using Flask-Testing without mocks.
+
+This module contains unit tests for the rest.py module, ensuring proper
+functionality of API request handling under various scenarios.
+"""
+
+import unittest
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)